feat(genapi): add reasoning effort troubleshooting section (#6407)
scaleway/docs-content: pages/generative-apis/troubleshooting/fixing-common-issues.mdx

  - For `mistral-nemo-instruct-2407	`, `temperature` should be set lower than `0.3`.
- Refer to the [documentation on structured outputs](/generative-apis/how-to/use-structured-outputs/) for examples and additional guidance.

## Reasoning model response is slow

### Description

> - When using a reasoning model such as `qwen3.5-397b-a17b` or `gpt-oss-120b` time to first token is slow (eg. more than `10` seconds), even when using `stream:true`.

Reviewer: ```suggestion
- When using a reasoning model such as `qwen3.5-397b-a17b` or `gpt-oss-120b`, the time to first token is slow (e.g., more than `10` seconds), even when using `stream:true`.
```

### Causes

> - Reasoning model is verbose and generate many reasoning tokens in `message.reasoning_content` before generating tokens in `message.content`.

Reviewer: ```suggestion
- The reasoning model is verbose and generates many reasoning tokens in `message.reasoning_content` before generating tokens in `message.content`.
```

> - Input tokens payload is significant and takes a long time to be processed.

Reviewer: ```suggestion
- The input tokens payload is significant and takes a long time to be processed.
```

### Solution

> - Disable reasoning by using `reasoning_effort: none` in request payload:

Reviewer: ```suggestion
- Disable reasoning by using `reasoning_effort: none` in the request payload:
```

  - The model will therefore not output reasoning tokens, and the field `reasoning` will be empty, leading to a faster answer.

>   - Note that in some edge cases, such as asking mathematical or logical questions, some models can still produce verbose responses, because of their default behaviour, even without activating "reasoning"/"thinking". As a workaround, we recommend editing your prompts, by asking the model to be brief. Example: `What is the solution to x⁴+x³+1=0 ? Be brief.` 

Reviewer: ```suggestion
  - Note that in some edge cases, such as asking mathematical or logical questions, some models can still produce verbose responses, because of their default behavior, even without activating "reasoning"/"thinking". As a workaround, we recommend editing your prompts, by asking the model to be brief. Example: `What is the solution to x⁴+x³+1=0 ? Be brief.` 
```

> - Optimize your request to reduce input tokens number in your payload.

Reviewer: ```suggestion
- Optimize your request to reduce the number of input tokens in your payload.
```
